packages/caliper-vscode/extension/.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
You should reuse the eslintrc file content from one of the other packages, like caliper-core, so it's in sync with the other packages